bigbluebutton-html5/imports/ui/components/video-provider/video-provider-graphql/component.tsx

```diff
@@ -411,33 +469,33 @@ class VideoProviderGraphql extends Component {
     }
   }
 
-  getStreamsToConnectAndDisconnect(streams) {
-    const streamsCameraIds = streams.filter(s => !s?.isGridItem).map(s => s.stream);
+  getStreamsToConnectAndDisconnect(streams: VideoItem[]) {
+    const streamsCameraIds = streams.filter((s) => s?.type !== 'grid').map((s) => (s as StreamItem).stream);
```

Comment:
is `type = 'grid'` replacing `isGridItem`? Do we still need both after the changes made in this PR?

Reply:
> is type = 'grid' replacing isGridItem? 

Yes!

> Do we still need both after the changes made in this PR?

No, actually. Nice catch.